cli/tsc/mod.rs

```diff
@@ -444,17 +441,29 @@ pub fn as_ts_script_kind(media_type: MediaType) -> i32 {
   }
 }
 
-// TODO(bartlomieju): `op2` doesn't support `serde_json::Value`
-#[op]
-fn op_load(state: &mut OpState, args: Value) -> Result<Value, AnyError> {
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+struct LoadResponse {
+  data: Option<String>,
```

Comment:
Not sure if this is correct - `op2` doesn't handle `Cow<'_, str>`, but I believe `op` macro did turn it into `String` internally anyway.

Reply:
I don't think this is an issue because it's used only in a serde struct.